plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -102,11 +103,13 @@ func (d *defaultServicePlan) Admit(a admission.Attributes) error {
 	// the order changes, we will need to rethink the
 	// implementation of this controller.
 
-	// implement field selector
-
 	// loop over all service plans accumulate into slice
```

Comment:
Maybe update this comment? get all the serviceplans for this class?

Reply:
sure!